libtorch_test/CMakeLists.txt

```diff
@@ -27,13 +23,15 @@ add_custom_target(
   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
 
 add_custom_target(
-  torch_test_soldout_wo_section
-  EXCLUDE_FROM_ALL
+  torch_test_soldout_wo_section ALL
+  # EXCLUDE_FROM_ALL
```

Comment:
Did you intentinoally include this to ALL?

Reply:
Ooops, just a mistake. Fixed in https://github.com/pfnet/sold/pull/2/commits/c826b7f7cdb2eca5d3e0abf6c4ea7a94e3d5c91b.